Update release-notes.md for 7.6.1

dtranquang opened the pull request:
<!-- 
Insert your contribution description here.
Ensure your contribution fills the Contributing Guidelines described here:
https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md 
-->

<!-- 
Special Attention to TITLE format
https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md#title
-->

Insert your Pull Request description here...

* add support for ...
* fix ... behaviour
* refactor ....

dtranquang committed:
Update release-notes.md

tbouffard reviewed (approved):
Seems ok.
I have checked with @abirembaut: we do not have any external contribution (engine, web, uid).

benjaminParisel committed:
remove BS in doc-site project

Doc-site is a tooling to build documentation and it's not necessary to notify fix in release-note

dtranquang committed:
add extra doc fixes

tbouffard commented:
@dtranquang the 7.6.1 version has been released for a while and we did not publish the release notes on the documentation site.
Can I merge this PR?

NathalieC committed:
doc(RN 7.6.1): fix some typos in 7.6.1 bug names
